cmake/app/boilerplate.cmake

```diff
@@ -236,13 +236,20 @@ DTC_OVERLAY_FILE=\"dts1.overlay dts2.overlay\"")
 set(CMAKE_C_COMPILER_FORCED   1)
 set(CMAKE_CXX_COMPILER_FORCED 1)
 
-include(${ZEPHYR_BASE}/cmake/GetGitRevisionDescription.cmake)
 include(${ZEPHYR_BASE}/cmake/version.cmake)
 include(${ZEPHYR_BASE}/cmake/host-tools.cmake)
 include(${ZEPHYR_BASE}/cmake/kconfig.cmake)
 include(${ZEPHYR_BASE}/cmake/toolchain.cmake)
 
-git_describe(BUILD_VERSION)
+if(NOT GIT_FOUND)
+  find_package(Git QUIET)
```

Comment:
I thought find_package would check for GIT_FOUND and return immediately if found already, making this if-statement unnecessary.